generator.py

```diff
@@ -78,102 +83,72 @@ def regen_crates(path, conf, level=0):
         if isdir(entry_file):
             if level < 2 and not regen_crates(entry_file, conf, level + 1):
                 return False
-        elif entry == 'Gir.toml':
+        elif fnmatch.fnmatchcase(entry, "Gir*.toml"):
```

Comment:
Seems a bit overkill compared to just `elif entry.startwith("Gir") and entry.endswith(".toml"):`...

Reply:
I would call it python style, but I don't care.